### PR TITLE
DOC-1685 3.9 Release note improvements

### PR DESCRIPTION
**Pull request by victorleeTG**

### CODE REVIEW

File: modules/release-notes/pages/index.adoc [hunk 1]
```diff
@@ -2,82 +2,98 @@
...
+[scalar accumulator] data types.
+* Added support for batch deletion of multiple edges between vertices, both with GSQL `DELETE` and as a xref:3.9@tigergraph-server:API:built-in-endpoints.adoc#_delete_an_edge_by_source_target_edge_type_and_discriminator[REST endpoint].
+* Standardized `SHOW PACKAGE` for xref:3.9@:graph-ml:using-an-algorithm.adoc#_packaged_template_queries[packaged queries] to follow the same behavior as other `SHOW` catalog commands.
 
 === JDBC Spark Connector
 
-* Added support for loading job monitoring on xref:data-loading:spark-connection-via-jdbc-driver.adoc[JDBC Spark connector] loading jobs.
-* Added the ability to assign a specific job ID to a JDBC loading job.
-* Added error limiting parameters to JDBC loading jobs to cancel loading automatically when errors surpass a certain threshold.
+* [Alex] Added support for loading job monitoring on xref:3.9@data-loading:spark-connection-via-jdbc-driver.adoc[JDBC Spark connector] loading jobs.
```
Comment (alexthomasTG) [inline modules/release-notes/pages/index.adoc:57]:
```suggestion
* Added support for loading job monitoring on xref:3.9@data-loading:spark-connection-via-jdbc-driver.adoc[JDBC Spark connector] loading jobs. This allows users to check loading progress as a job is running, letting them catch errors before a job is completed.
```

File: modules/release-notes/pages/index.adoc [hunk 2]
```diff
@@ -2,82 +2,98 @@
...
+* [Alex] Added the ability to assign a specific job ID to a JDBC loading job.
```
Comment (alexthomasTG) [inline modules/release-notes/pages/index.adoc:58]:
```suggestion
* Added the ability to assign a specific job ID to a JDBC loading job in order to manually control each job individually.
```

File: modules/release-notes/pages/index.adoc [hunk 3]
```diff
@@ -2,82 +2,98 @@
...
+* [Alex] Added error limiting parameters to JDBC loading jobs to cancel loading automatically when errors surpass a certain threshold.
```
Comment (alexthomasTG) [inline modules/release-notes/pages/index.adoc:59]:
```suggestion
* Added error limiting parameters to JDBC loading jobs to cancel loading automatically when errors surpass a certain threshold.
```

File: modules/release-notes/pages/index.adoc [hunk 4]
```diff
@@ -2,82 +2,98 @@
...
+LTS versions are supported for 24 months from their initial release and should be the choice for production deployments.
 
-The following is a list of new features and improvements:
+
+
+== Key New Features
+
+* *xref:3.9@gsql-ref:ddl-and-loading:defining-a-graph-schema.adoc#_discriminator[Multiple edges of the same type between vertices]* - Simplifies application design for time series data and other complex large data sets
+* *xref:3.9@tigergraph-server:data-loading:data-streaming-connector/[Data streaming connector]* - Simplifies and unifies setup for data loading, employs robust Kafka-based design, supports cloud-hosted data sources in many formats, including Parquet
+* *Improved monitoring and logging* - [Alex] Facilitates better workload management for xref:3.9@:gui-docs:monitoring:queries.adoc[queries] and loading jobs
```
Comment (alexthomasTG) [inline modules/release-notes/pages/index.adoc]:
```suggestion
* *Improved monitoring and logging* - Facilitates better workload management for xref:3.9@:gui-docs:monitoring:queries.adoc[queries] and xref:3.9@:gsql-ref:ddl-and-loading:managing-loading-job.adoc[loading jobs].
```

File: modules/release-notes/pages/index.adoc [hunk 5]
```diff
@@ -2,82 +2,98 @@
...
+* [Alex] Added the ability to assign a specific job ID to a JDBC loading job.
+* [Alex] Added error limiting parameters to JDBC loading jobs to cancel loading automatically when errors surpass a certain threshold.
 
 === Kubernetes Operator
 
-* Expanded the functionality of the xref:kubernetes:k8s-operator/index.adoc[].
-The Operator is now the preferred way for users to perform cluster operations with Kubernetes.
-* Added xref:kubernetes:k8s-operator/backup-and-restore.adoc[], version upgrade, and resource update functionality.
+* Enhanced the functionality of the xref:3.9@kubernetes:k8s-operator/index.adoc[] so it is now the standard way to perform cluster operations with Kubernetes.
+* Added xref:3.9@kubernetes:k8s-operator/backup-and-restore.adoc[], [Alex] version upgrade, and [Alex] resource update functionality.
```
Comment (alexthomasTG) [inline modules/release-notes/pages/index.adoc]:
```suggestion
* Added xref:3.9@kubernetes:k8s-operator/backup-and-restore.adoc[], xref:3.9@kubernetes:k8s-operator/cluster-operations.adoc#upgrade-cluster-version[version upgrade], and xref:3.9@kubernetes:k8s-operator/cluster-operations.adoc#update-cluster-resources[resource update] functionality.
```

**Review by alexthomasTG — APPROVED** [review]

File: modules/release-notes/pages/index.adoc (hunk 1 above)
Comment (victorleeTG) [inline modules/release-notes/pages/index.adoc]: @alexthomasTG What does a User need to do to monitor the progress?  The referenced page doesn't mention monitoring. Would it be better to link to a different page that does address monitoring?

File: modules/release-notes/pages/index.adoc (hunk 2 above)
Comment (victorleeTG) [inline modules/release-notes/pages/index.adoc]: @alexthomasTG Where in the documentation do we show how to assign an ID?

File: modules/release-notes/pages/index.adoc (hunk 3 above)
Comment (victorleeTG) [inline modules/release-notes/pages/index.adoc]: @alexthomasTG Is this a user-configurable threshold? Where is this parameter described?

File: modules/release-notes/pages/index.adoc (hunk 5 above)
Comment (victorleeTG) [inline modules/release-notes/pages/index.adoc]:
```suggestion
* Added support for xref:3.9@kubernetes:k8s-operator/backup-and-restore.adoc[], xref:3.9@kubernetes:k8s-operator/cluster-operations.adoc#upgrade-cluster-version[upgrading the version of the TigerGraph server], and xref:3.9@kubernetes:k8s-operator/cluster-operations.adoc#update-cluster-resources[updating cluster resources].
```

File: modules/release-notes/pages/index.adoc [hunk 6]
```diff
@@ -2,82 +2,98 @@
 :description: Release notes for TigerGraph {page-component-version}.
```
Comment (victorleeTG) [inline modules/release-notes/pages/index.adoc]:
```suggestion
:description: Release notes for TigerGraph {page-component-version} LTS.
```

File: modules/release-notes/pages/index.adoc [hunk 7]
```diff
@@ -2,82 +2,98 @@
...
 :page-aliases: change-log.adoc, release-notes.adoc
 :fn-preview: footnote:preview[Features in the preview stage should not be used for production purposes. General Availability (GA) versions of the feature will be available in a later release.]
+:toc:
+:toclevels:2
 
 TigerGraph Server {page-component-version}.0 was released on February 28th, 2023.
```
Comment (victorleeTG) [inline modules/release-notes/pages/index.adoc]:
```suggestion
TigerGraph Server {page-component-version}.0 LTS was released on February 28th, 2023.
```

**Pull request merged**